docs/playground/index.asciidoc

```diff
@@ -93,22 +93,25 @@ a|
 [[playground-getting-started]]
 == Getting started
 
+[.screenshot]
+image::get-started.png[width=600]
+
 [float]
 [[playground-getting-started-connect]]
 === Connect to LLM provider
 
 To get started with {x}, you need to create a <<action-types,connector>> for your LLM provider.
 Follow these steps on the {x} landing page:
 
-. Under *Connect to LLM*, click *Create connector*.
+. Under *Connect to an LLM*, click *Create connector*.
 . Select your *LLM provider*.
 . *Name* your connector.
 . Select a *URL endpoint* (or use the default).
 . Enter *access credentials* for your LLM provider.
 
 [TIP]
 ====
-If you need to update a connector, or add a new one, click the wrench button (🔧) under *Model settings*.
```

Comment:
could we also mention how to add an additional LLM connector? The easiest way is to go to connectors and add one (either gemini or bedrock or openai). Once added, they will show up in the modal list.

Reply:
worth linking to the security team's guide on adding connectors 
Bedrock for example: https://www.elastic.co/docs/current/serverless/security/connect-to-bedrock

Reply:
I'll circle back in follow-up for this 👍